ITR/data/base_providers.py

```diff
@@ -664,7 +667,8 @@ def _get_trends(self, intensities: pd.DataFrame):
             # ratios are dimensionless, so get rid of units, which confuse rolling/apply.  Some columns are NaN-only
             intensities[col] = intensities[col].map(lambda x: x.m if isinstance(x, pint.Quantity) else (breakpoint (), x))
             # FIXME: rolling windows require conversion to float64.  Don't want to be a nuisance...
-            intensities[col] = unp.nominal_values(intensities[col])
+            if ITR.HAS_UNCERTAINTIES:
+                intensities[col] = ITR.nominal_values(intensities[col])
```

Comment:
we loose `intensities` here

Reply:
nominal_values just picks off the nominal values of UFloats.  If itensities[col] only holds np.float64 values, they are already at nominal values and nothing need be done.